pkgs/development/python-modules/splinter/default.nix

```diff
@@ -24,21 +33,28 @@ buildPythonPackage rec {
   ];
 
   checkInputs = [
+    cssselect
+    django
     flask
+    lxml
     pytestCheckHook
+    zope-testbrowser
   ];
 
   disabledTestPaths = [
     "samples"
-    "tests/test_djangoclient.py"
-    "tests/test_flaskclient.py"
+    "tests/test_async_finder.py"
```

Comment:
Would be nice to include some reasoning for disabling all these else it just looks like we're disabling anything that fails.